cmd/kubelet/app/server.go

```diff
@@ -226,7 +231,7 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 			// Config and flags parsed, now we can initialize logging.
 			logs.InitLogs()
 			logOption := &logs.Options{Config: kubeletConfig.Logging}
-			if err := logOption.ValidateAndApply(); err != nil {
+			if err := logOption.ValidateAndApply(utilfeature.DefaultFeatureGate); err != nil {
```

Comment:
this works because it is *after* `utilfeature.DefaultMutableFeatureGate.SetFromMap(kubeletConfig.FeatureGates)` and after `InitLogs()`